src/tabs/presets/presets.css

```diff
@@ -125,6 +125,11 @@
     color: var(--defaultText);
 }
 
+.presets_filter_select_nonempty {
+    border-color: var(--accentBorder) !important;
+    border-width: 2px !important;
```

Comment:
😆 does it have to  be `!important` usually this will lead to more headscrathing than solved problems

Reply:
This is a sort of hacking for multiSelect jquery component styling. The original styles and html are defined within the component.